compiler/damlc/daml-preprocessor/src/DA/Daml/Preprocessor.hs

```diff
@@ -48,12 +50,21 @@ damlPreprocessor mbPkgName x
     | otherwise = IdePreprocessedSource
         { preprocWarnings = checkModuleName x
         , preprocErrors = checkImports x ++ checkDataTypes x ++ checkModuleDefinition x
-        , preprocSource = recordDotPreprocessor $ importDamlPreprocessor $ genericsPreprocessor mbPkgName $ templateConstraintPreprocessor x
+        , preprocSource = recordDotPreprocessor $ importDamlPreprocessor $ genericsPreprocessor mbPkgName $ templateConstraintPreprocessor $ enumTypePreprocessor x
         }
     where
       name = fmap GHC.unLoc $ GHC.hsmodName $ GHC.unLoc x
 
--- | No preprocessing. Used for generated code.
+-- | Preprocessor for generated code.
+genPreprocessor :: GHC.ParsedSource -> IdePreprocessedSource
```

Comment:
I think it would be useful to spell this out to `generatedPreprocessor`. Otherwise it is very easy to confuse this with the `genericsPreprocessor`.

Reply:
done, thanks!